src/views/Generator/RuleEditor/RuleEditor.tsx

```diff
@@ -113,6 +104,10 @@ export function RuleEditor({ rule }: RuleEditorProps) {
               <ChevronLeftIcon />
               Back
```

Comment:
Not sure about the button placement. Currently, it's on the left. Do you think it should be on the right instead?

Reply:
I think it looks good on the left 🤔